packages/python/plotly/plotly/express/_core.py

```diff
@@ -1499,7 +1498,9 @@ def aggfunc_discrete(x):
 
         if args["color"]:
             if args["color"] == args["values"]:
-                aggfunc_color = "sum"
+                new_value_col_name = args["values"] + "_total"
+                df[new_value_col_name] = df[args["values"]]
+                args["values"] = new_value_col_name
```

Comment:
so like this, the `<col>_total` appears in the hoverlabel as the value. This is what we want? Not sure what would be better...

Reply:
Maybe we want `<col>_sum` instead? or `<col>_weighted_average` for the other one? not sure

Reply:
`<col>_color` for the other one maybe?

Reply:
I'm fine changing `total` to `sum`. I put `total` like `branchvalues='total'`. We could also put a suffix for the `colors` column but if we want to minimize the occurrence of such suffixes, we'd better put it on the `values` column (only in the case where it's the same as the `colors` column).

Reply:
ok let's do `sum` and merge this thing :)